### ISSUE TITLE
Search results always contains just one hit

### ISSUE THREAD

**Issue by steinm** [pydio/cells#373]
Not matter how I search, the result has just one hit
![2021-11-16_11-43](https://user-images.githubusercontent.com/1769107/141971017-87779d17-b5fa-4bdd-b0a0-a5b189a6e3c9.png)

The filter settings indicate that there are way more hits, but I've never seen more than single hit.
I'm using 3.0.1

**Comment by steinm** [comment]
I took the cells sdk v3, implemented a simple search command and got the same result. Whatever I search for, the result set has always just one entry and the facets indicate that there are many more. I even set the parameter 'Size' to 10, which is the default anyway (I suppose). Here is the code snipped I use

```var (
    query string
)

var searchNodes = &cobra.Command{
    Use: "search",
    Long: `Search nodes in your Pydio Cells server.
    
    Example:
    $ go run main.go -u https://pydio.example.com -l admin -p password -f common-files/*  -q term search 
    `,
    Run: func(cmd *cobra.Command, args []string) {

        // Connect to the pydio api via the sdkConfig
        ctx, apiClient, err := rest.GetClient(DefaultConfig, false)
        if err != nil {
            log.Fatal(err)
        }

        params := &search_service.NodesParams{
            Body: &models.TreeSearchRequest{Details: true, Query: &models.TreeQuery{FileName: query}, Size: 10},
            Context: ctx,
        }

        // Assign the files data retrieved above to the result variable
        result, err := apiClient.SearchService.Nodes(params)
//        fmt.Printf("%+v", result.Payload);
        if err != nil {
            fmt.Printf("Could not find nodes\n")
            log.Fatal(err)
        }

```

Any idea?

**Comment by cdujeu** [comment]
Hello @steinm sorry for the late answer. It would have been better to post in the [forum](https://forum.pydio.com) until we qualify a proper issue for github. Did you find anything useful since then?

**Comment by steinm** [comment]
Just checked again, but the behaviour is still unchanged.

**Comment by cdujeu** [comment]
Can you try to fully re-index the search engine by launching the following command :
`./cells admin resync --service=pydio.grpc.search --path=/`

(cells server must be running)

**Comment by steinm** [comment]
Just did that and I got a 'Resync Triggered' which looks like it should be doing something.

Is that './cells admin resync' the same as starting a manual resync for a datasource in the web gui? Because, that's what I did already this morning and it's telling me that the last sync was done an hour ago. The search still just returns one entry.

What really puzzles me is the fact that the search appears to find more files and folders, but doesn't output them.
